Automattic/wp-calypso
# Add/domains prototype

THIS IS NOT PRODUCTION CODE

This is just a prototype to play with some ideas for how /domains could work. These ideas are subject to change. The technical solutions aren't well thought through or final.

Technical considerations for the final implementation
- Split to bundle at /domains so that the logged out bundle is smaller
- Make the site-selected signup flow work and disable the blog option
- Save the domain in the site title

### How to review the flows
Kudos to @drewblaisdell for instruction, this seems to be generally how you test this:

1. Visit [/start/domain-first](http://calypso.localhost:3000/start/domain-first) while logged out.
2. Select a domain.
3. Complete signup for a new account.
4. Purchase the domain.
5. You should be redirected to /domains/manage/:site. You're in the My Site section of Calypso, but there are no links on the sidebar except to the sitepicker.
6. Clicking Set Up Site takes you to a special signup flow to set up your site (i.e. turn it from a domain-only site into a normal site). As far as I can tell, this flow does not work yet.

Questions etc belong on P2...
